src/modules/auth/components/EnterPasswordForm/enterPasswordForm.test.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+import useDecryptionAccount from '@account/hooks/useDecryptionAccount';
+import EnterPasswordForm from '.';
+import styles from './enterPasswordForm.css';
+
+jest.mock('@account/hooks/useDecryptionAccount');
+
+describe('PassphraseBackup', () => {
+  let wrapper;
+  const props = {
+    accountSchema: {
+      metadata: {
+        address: 'lskm555k7nhhw954rw4pqy5q9wn28n3cec94fmp4n',
+        name: 'Lisker',
+      },
+    },
+    onEnterPasswordSuccess: jest.fn(),
+  };
+
+  it('should display properly', () => {
+    wrapper = mount(<EnterPasswordForm {...props} />);
+
+    expect(wrapper.find('.accountName')).toHaveText(props.accountSchema.metadata.name);
+    expect(wrapper.find('.accountAddress')).toHaveText(props.accountSchema.metadata.address);
+  });
+
+  it('should call onEnterPasswordSuccess when onSubmit click', () => {
+    const privateToken = 'private-token-mock';
+    const recoveryPhrase = 'target cancel solution recipe vague faint bomb convince pink vendor fresh patrol';
+    useDecryptionAccount.mockImplementation(() => (
+      {
+        privateToken,
+        recoveryPhrase,
+      }
+    ));
+    wrapper = mount(<EnterPasswordForm {...props} />);
+
+    wrapper.find('input').at(0).simulate('change', {
+      target: {
+        value: 'qwerty',
+      },
+    });
+    wrapper.find(styles.button).first().simulate('click');
```

Comment:
dear, could you please change the find method,
in front-end, we will change the styles often so it's will end up with a broken test

Reply:
Done. No need to call me dear.